apps/server/src/app/app.module.ts

```diff
@@ -40,6 +40,7 @@ import { AppService } from './app.service';
     UsersModule,
     TasksModule,
     CategoriesModule,
+    ScheduleModule.forRoot(),
```

Comment:
What do we need `.forRoot()` for?

Reply:
According to the documentation: _The .forRoot() call initializes the scheduler and registers any declarative [cron jobs](https://docs.nestjs.com/techniques/task-scheduling#declarative-cron-jobs), [timeouts](https://docs.nestjs.com/techniques/task-scheduling#declarative-timeouts) and [intervals](https://docs.nestjs.com/techniques/task-scheduling#declarative-intervals) that exist within your app. Registration occurs when the onApplicationBootstrap lifecycle hook occurs, ensuring that all modules have loaded and declared any scheduled jobs._

Reply:
Can we add that as a comment above the line adding `forRoot`?